usaspending_api/api_contracts/contracts/v2/search/spending_by_subaward_grouped.md

```diff
@@ -0,0 +1,258 @@
+FORMAT: 1A
+HOST: https://api.usaspending.gov
+
+# Spending by Award [/api/v2/search/spending_by_subaward_grouped/]
+
+This endpoint supports the advanced search page and allow for complex filtering for specific subsets of spending data.
+
+## POST
+
+Searches for subaward records based on a provided set of filters and groups them by their prime award.
+
++ Request (application/json)
+    + Schema
+
+            {
+                "$schema": "http://json-schema.org/draft-04/schema#",
+                "type": "object"
+            }
+
+    + Attributes (object)
+        + `filters` (required, AdvancedFilterObject)
+            These filters are applied at the Subaward level (`children`). They will not affect the Prime Awards (`results`) themselves
+        + `fields` (required, SpendingBySubawardFields)
+            The field names to include for `children` objects (Subawards) in the response.
+        + `limit`: 5 (optional, number)
+            The number of `results` (Prime Awards) to include per page. The number of `children` (Subawards) objects will always be limited to 10
```

Comment:
The designers added the ability to change the # of rows of Prime Awards to show.  I can ask them to remove that if we're limited to 10
![Screenshot 2024-04-19 at 10 20 53 AM](https://github.com/fedspendingtransparency/usaspending-api/assets/121010/0a3bb17a-07b1-4cd9-8d42-b7c5c3d006bd)

Reply:
Hey Andrea, the Prime Award level, shown here, can be adjusted with the `limit` field. For changes in number of results shown at the Subaward (child) level, the Spending by Award endpoint can be used. This is a similar pattern for the Spending by Transactions view/endpoint.

Reply:
Hey Andrea, I added the missing columns to the contract. Also left a comment in the ticket containing a mapping of columns -> API field -> database field.